apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-body.tsx

```diff
@@ -36,7 +36,13 @@ export const InAppBody = () => {
               />
             </InputRoot>
           </FormControl>
-          <FormMessage>{`Type {{ for variables, or wrap text in ** for bold.`}</FormMessage>
+          <FormMessage>
+            {containsHTMLEntities(field.value) && !getValues('disableOutputSanitization')
+              ? 'HTML entities detected. Consider disabling content sanitization for proper rendering'
+              : field.value.length > 2 && !containsVariables(field.value)
```

Comment:
Plus another nice touch to show the variable hint after 2 characters are typed, and if no other variables are contained.